pkg/apimachinery/utils/meta.go

```diff
@@ -718,6 +747,107 @@ func (m *grafanaMetaAccessor) FindTitle(defaultTitle string) string {
 	return defaultTitle
 }
 
+func (m *grafanaMetaAccessor) GetManagerProperties() (ManagerProperties, bool) {
+	res := ManagerProperties{
+		Identity:    "",
+		Kind:        ManagerKindUnknown,
+		AllowsEdits: true,
```

Comment:
```suggestion
		AllowsEdits: false,
```
IIUC

Reply:
No, we want to NOT block edits by default, which is why we fall back to `true`, i.e. in the storage strategy (or other middleware) we'd be checking if the requester identity matches with that of the annotations identity and if there's a mismatch, we'd inspect whether this flag is set to true. If it is, we'd allow the edit, otherwise forbid.